dist/index.d.ts

```diff
@@ -1,8 +1,10 @@
 declare const makeErrorChecker: (e: object) => {
+    [x: string]: boolean | object;
```

Comment:
Did you add this here to make TS checks pass? I'm guessing because TS can't handle the `Symbol` type. Hmm. I'd rather remove this (manually if need be).